app/assets/stylesheets/_mixins.scss

```diff
@@ -0,0 +1,11 @@
+@mixin shadow-box($amount, $color) {
+  box-shadow: inset 0 0 0 $amount $color;
+  -moz-box-shadow: inset 0 0 0 $amount $color;
+  -webkit-box-shadow:inset 0 0 0 $amount $color;
+}
+
+@mixin cell-gradient($from, $to) {
+  background:#cecece;
+  background:-moz-linear-gradient(top, $from, $to);
+  background:-webkit-gradient(linear,0 0, 0 100%, from($from), to($to));
+}
```

Comment:
All files should end in a newline. POSIX spec defines a line as ending in a carriage return:

http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206
http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline